Entities/Industry/CTFShops/BuilderShop/BuilderShop.as

```diff
@@ -52,6 +52,13 @@ void onInit(CBlob@ this)
 		s.buttonheight = 1;
 		AddRequirement(s.requirements, "blob", "mat_stone", "Stone", CTFCosts::boulder_stone);
 	}
+	{
+		ShopItem@ s = addShopItem(this, "Boulder", "$boulder$", "boulder", Descriptions::boulder, false);
+		s.customButton = true;
+		s.buttonwidth = 2;
+		s.buttonheight = 1;
+		AddRequirement(s.requirements, "blob", "mat_stone", "Stone", CTFCosts::boulder_stone);
+	}
```

Comment:
There shouldn't be this duplicate boulder entry.

Reply:
I... don't even know how that got in, it doesn't even appear in the shop xD